sdk/appconfiguration/app-configuration/test/index.readonlytests.spec.ts

```diff
@@ -58,9 +61,14 @@ describe("AppConfigurationClient (set|clear)ReadOnly", () => {
       409,
       "Delete should fail because the setting is read-only"
     );
+
+    await deleteKeyCompletely([testConfigSetting.key], client);
```

Comment:
@richardpark-msft Remind me, you intended for deleteKeyCompletely to be done in the `after` function right? Maybe this should go in the `afterEach` now? I think that way if one of the tests fail for some reason, the resource still gets cleaned up.

Reply:
Do we need to have these in the afterEach? I believe the recorder might not support network calls outside of it() calls at the moment. I believe I was having issues with it, and thus I moved this to the individual tests. I can try again if having this in the afterEach is preferred.

Reply:
I was sable to move it to the afterEach!